package.json

```diff
@@ -6,20 +6,26 @@
   "dependencies": {
     "core-util-is": "~1.0.0",
     "inherits": "~2.0.1",
+    "inline-process-browser": "~2.0.1",
     "isarray": "~1.0.0",
     "process-nextick-args": "~1.0.6",
     "string_decoder": "~0.10.x",
+    "unreachable-branch-transform": "~0.5.0",
```

Comment:
We need a way to force "readable-stream" to use node own stream module. We need this to verify new streams implementations do not break existing modules before releasing new versions of node.
This feature increases the browserify bundle by a huge factor, so the two transform for browserify were included.
I'll be more than happy to have an alternative implementation that achieve both goals.